### DISCUSSION TITLE
Calendar.html with condition to show month view or week view

### DISCUSSION THREAD

**Discussion by averlon** [derhansen/sf_event_mgt#1077] [Ideas]
Hi,
I was trying to show in FE a month view and on request a week view.
I then found out, the Calender.html only supports one view at a time.

So I went the following way, and probably, this is something that could be used in general - not sure:
1) TSconfig:
`tx_sfeventmgt.templateLayouts {
  1 = viewyear
  2 = viewmonth
}`

Calender.html with condition included (see attached).

In addition:
1) I downloaded the de.locallang.xml from crowdin. Result: Calendar is now shown in FE in DE-Language since this is the standard in my config.
2) I added a translation to "week", since this is still hard coded in the Calendar.html
3) I added some fluid translate into the Calendar.html for the week view
Comment: unfortunately this does not work so far - where I don't know why not.

[Calendar.txt](https://github.com/derhansen/sf_event_mgt/files/10732312/Calendar.txt)

**Comment by derhansen** [comment]
I will not add any default template layouts to the extension. This must always be done manually by the integrator in each TYPO3 website.

I will add a localization string for the word `week` in the default template for `Calendar.html` of the extension (v6 and v7 only)

**Reply by averlon** [reply]
O.K. - you are the owner!
Although Calendar.html is a default template - as far as I understand T3 and fluid. Some enhancements to create a month view and a week view would help. Alternatively a new STANDARD template for the week view could be created and distributed so the user has a choice to select via the pluging config.
your decision!
